models/user.rb

```diff
@@ -10,9 +10,12 @@ class User < Base
 
   RESET_WINDOW = 60 * 15 # 15 minutes
 
-  SETTINGS = %w[
-    notifications bg font bg2 font2 red_logo white yellow green brown gray red blue
-  ].freeze
+  SETTINGS = 4.times.flat_map do |n|
+    %w[color dash width].map do |prop|
+      "r#{n}_#{prop}"
+    end
+  end
+  SETTINGS.concat(%w[notifications red_logo bg font bg2 font2 white yellow green brown gray red blue]).freeze
```

Comment:
you're mutating a constant here SETTINGS.concat mutates

Reply:
so
SET1 = 4.times …
SET2 = %w[…]
SETTINGS = SET1 + SET2

… or sth else?

Reply:
Settiings = 4.times... + %w[...]